src/client/components/PageHeaderComponent/PageHeader.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './PageHeader.styles.css';
+
+export default function PageHeader() {
+  return (
+    <header>
+      <h1>GuideIT</h1>
```

Comment:
I think, we will need one more variable textbox just upon the header. Please check the following screenshot :) @sofiiadidovych 

<img width="455" alt="Capture" src="https://user-images.githubusercontent.com/39958295/115150889-9dd6a700-a06a-11eb-82b1-229554b171bd.PNG">

Reply:
Tithi, I am not sure, I thought that is just a label for the pages to mark them in figma?

Reply:
Yes, right. It could be. I will ask Daniel in our channel to be in the safe side, you know :)

Reply:
Hi again. You are right about the design :) It looks perfect.

Reply:
Thank you, Tithi! that's great, that now we know for sure :)

Reply:
I have seen your code its looking good.

Reply:
Thank you, Jyothi.

Reply:
for me its seems like header height is more ,so may be can you check and reduce the height of the header.
height:50px;

Reply:
Thank you, Jyothi. I will try to figure out something, but I intensionally avoided using px and used em: padding: 0.1em; in order to keep it responsive.